source/user/pbc/gto.rst

```diff
@@ -0,0 +1,167 @@
+.. _user_pbc_gto:
+
+Crystal structure
+*****************
+
+*Modules*: :mod:`pbc.gto`
+
+Initializing a crystal
+======================
+
+Initializing a crystal unit cell is very similar to the initialization
+of a molecule. Instead of the :class:`gto.Mole` class, 
+one uses the :class:`pbc.gto.Cell` class to define a cell::
+
+  >>> from pyscf.pbc import gto
+  >>> cell = gto.Cell()
+  >>> cell.atom = '''H  0 0 0; H 1 1 1'''
+  >>> cell.basis = 'gth-dzvp'
+  >>> cell.pseudo = 'gth-pade'
+  >>> cell.a = numpy.eye(3) * 2
+  >>> cell.build()
+
+The other two equivalent ways to initialize a molecule 
+introduced in :numref:`user_gto` also apply here::
+
+  >>> from pyscf.pbc import gto
+  >>> cell = gto.Cell()
+  >>> cell.build(
+  ...   atom = '''H  0 0 0; H 1 1 1''',
+  ...   basis = 'gth-dzvp',
+  ...   pseudo = 'gth-pade',
+  ...   a = numpy.eye(3) * 2)
+
+  >>> import pyscf
+  >>> cell = pyscf.M(
+  ...   atom = '''H  0 0 0; H 1 1 1''',
+  ...   basis = 'gth-dzvp',
+  ...   pseudo = 'gth-pade',
+  ...   a = numpy.eye(3) * 2)
+
+  >>> from pyscf.pbc import gto
+  >>> cell = gto.M(
+  ...   atom = '''H  0 0 0; H 1 1 1''',
+  ...   basis = 'gth-dzvp',
+  ...   pseudo = 'gth-pade',
+  ...   a = numpy.eye(3) * 2)
+
+
+Lattice vectors
+---------------
+
+The crystal initialization requires an additional parameter :attr:`~Cell.a`, which
+represents the lattice vectors. The format of :attr:`~Cell.a` is array-like::
+    
+    cell.a = numpy.eye(3) * 2
+    cell.a = [[2,0,0],[0,2,0],[0,0,2]]
+
+Each row of the 3-by-3 matrix of :attr:`~Cell.a` represents a lattice vector
+in Cartesian coordinate, with the same unit as the input :attr:`atom` parameter.
+
+.. note:: It is recommended that the input lattice vectors (row by row) form a 
+          right-handed coordinate system.
```

Comment:
What happens if they don't form a rhs coordinate system?

Reply:
```suggestion
.. note:: The input lattice vectors (row by row) should form a right-handed coordinate system, as otherwise some integrals may be computed incorrectly in PySCF.
```